gridstatus/spp.py

```diff
@@ -1139,6 +1141,137 @@ def _process_operating_reserves(self, df):
 
         return df
 
+    def get_as_prices_real_time_5_min(
+        self,
+        date: str | pd.Timestamp,
+        end: str | pd.Timestamp | None = None,
+        verbose: bool = False,
+        use_daily_files: bool = False,
+    ):
+        """
+        Provides Marginal Clearing Price information by Reserve Zone for each
+        Real-Time 5-minute Market solution.
+
+        Args:
+            date: date to get data for. Supports "latest" for most recent interval.
+            end: end date
+            verbose: print url
+            use_daily_files: if True, use daily files instead of 5 minute files.
+
+        Returns:
+            pd.DataFrame: Real-Time 5-minute Marginal Clearing Prices
+        """
+        if use_daily_files:
+            return self._get_as_prices_real_time_5_min_from_daily_files(
+                date,
+                end=end,
+                verbose=verbose,
+            )
+        else:
+            return self._get_as_prices_real_time_5_min_from_intervals(
+                date,
+                end=end,
+                verbose=verbose,
+            )
+
+    @support_date_range("5_MIN")
+    def _get_as_prices_real_time_5_min_from_intervals(
+        self,
+        date,
+        end=None,
+        verbose=False,
+    ):
+        """Get AS prices from 5-minute interval files."""
+        if date == "latest":
+            url = f"{FILE_BROWSER_DOWNLOAD_URL}/rtbm-mcp?path=/RTBM-MCP-latestInterval.csv"
+        else:
+            url = self._format_5_min_url(
+                date,
+                end,
+                endpoint=RTBM_MCP,
+                file_prefix="RTBM-MCP",
+                include_interval=True,
+            )
+
+        log(f"Downloading {url}", verbose)
+        df = pd.read_csv(url)
+        return self._process_as_prices_real_time(df)
+
+    @support_date_range("DAY_START")
+    def _get_as_prices_real_time_5_min_from_daily_files(
+        self,
+        date,
+        end=None,
+        verbose=False,
+    ):
```

Comment:
The daily files are available on a 5-day delay from the current date.

<img width="2662" height="943" alt="Image" src="https://github.com/user-attachments/assets/477fa076-b5a5-4611-8b24-6217874065e9" />